### ISSUE TITLE
I am getting error Class 'Sly\NotificationPusher\Adapter\' not found when i use in lumen.

### ISSUE THREAD

**Issue by piyushpatil027** [davibennun/laravel-push-notification#81]
Hi,

Hope you are doing great. Your package are great . I used in Laravel 4but when i use in Lumen then i am  getting  [ Class 'Sly\NotificationPusher\Adapter\' not found ] this type of error. Please tell me where i am doing wrong. I google it but i did not got proper answer.

Please reply me as possible as.

**Comment by rahul0000** [comment]
Hello,
Is this problem solved? whats the solution.

Thanks.

**Comment by piyushpatil027** [comment]
No.

On Mon, Dec 14, 2015 at 11:24 AM, Rahul Jain <notifications@github.com>
wrote:

> Hello,
> Is this problem solved? whats the solution.
>
> Thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/davibennun/laravel-push-notification/issues/81#issuecomment-164348410>
> .
>



-- 
Thanks & Regards
Piyush Patil


Software Engineer
Sky Id : piyushpatil27
Email Id : piyushpatil027@gmail.com
Mobile No : 09096871715

**Comment by piyushpatil027** [comment]
I am use  directly Push Manager.

On Mon, Dec 14, 2015 at 12:21 PM, piyush patil <piyushpatil027@gmail.com>
wrote:

> No.
>
> On Mon, Dec 14, 2015 at 11:24 AM, Rahul Jain <notifications@github.com>
> wrote:
>
>> Hello,
>> Is this problem solved? whats the solution.
>>
>> Thanks.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/davibennun/laravel-push-notification/issues/81#issuecomment-164348410>
>> .
>>
>
>
>
> --
> Thanks & Regards
> Piyush Patil
>
>
> Software Engineer
> Sky Id : piyushpatil27
> Email Id : piyushpatil027@gmail.com
> Mobile No : 09096871715
>



-- 
Thanks & Regards
Piyush Patil


Software Engineer
Sky Id : piyushpatil27
Email Id : piyushpatil027@gmail.com
Mobile No : 09096871715

**Comment by computerfr33k** [comment]
I get this error as well, I did some debugging and found that the $config variable in the App.php constructor is actually null. I am using Laravel 5.2 and seems it doesn't work now.

**Comment by callam** [comment]
My guess is you have the dynamic configuration instructions. Try creating a configuration file called push-notification.php in your config and place ```$app->configure('push-notification')``` somewhere in your bootstrap.php.

push-notification.php

```
return [
    'appNameIOS' => [
        'environment' => 'development',
        'certificate' => '/path/to/certificate.pem',
        'passPhrase'  => 'password',
        'service' => 'apns'
    ],
];
```

**Comment by kaankilic** [comment]
the problem solved? I've got an same error on lumen.

**Comment by piyushpatil027** [comment]
Yes problem is solved . Use default "Sly\NotificationPusher\Adapter\"
package.



On Wed, Jan 20, 2016 at 2:46 AM, Kaan Kılıç <notifications@github.com>
wrote:

> the problem solved? I've got an same error on lumen.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/davibennun/laravel-push-notification/issues/81#issuecomment-172989476>
> .
>



-- 
Thanks & Regards
Piyush Patil


Software Engineer
Sky Id : piyushpatil27
Email Id : piyushpatil027@gmail.com
Mobile No : 09096871715

**Comment by younes0** [comment]
@callam is right.
In your bootstrap\app.php file, put this in your "Register Service Provider" section:

```php
$app->register(Davibennun\LaravelPushNotification\LaravelPushNotificationServiceProvider::class);

$app->configure('push-notification', [
    'appNameIOS' => [
        'environment' => 'development',
        'certificate' => '/path/to/certificate.pem',
        'passPhrase'  => 'password',
        'service'     => 'apns'
    ],
]);
``

**Comment by tvillafane** [comment]
Try setting the result of the PushNotification::app() method to a variable, then use it to send the message:

`$push = PushNotification::app([
	'environment' => 'development',
        'certificate' => app()->path() . YOUR_PATH,
        'passPhrase'  => 'YOUR_PASSPHRASE',
        'service'     => 'apns'
    ]);

 $push->to("YOUR_DEVICE")->send("TINY RICK!");`
